config/resources/parliament-domain.json

```diff
@@ -91,6 +103,44 @@
         }
       },
       "new-resource-base": "http://themis.vlaanderen.be/id/parlementaire-indieningsactiviteit/"
+    },
+    "submitted-pieces": {
+      "name": "submitted-piece",
+      "class": "parl:IngediendStuk",
+      "attributes": {
+        "unsigned-file": {
+          "type": "uri",
```

Comment:
Just for future reference: we did this to avoid `mu-cl-resources` throwing errors/omitting results when a file would get deleted after we've submitted it to the parliament. Do we want to make this clear somewhere in comments/documentation? Note to self here: add a vp-model to https://github.com/kanselarij-vlaanderen/kaleidos-documentation/tree/master/data-model that clarifies all this so we don't forget why we did it.